_conferences/droidcon-uk-2020.md

```diff
@@ -1,12 +1,12 @@
 ---
 name: "Droidcon"
-website: http://uk.droidcon.com/
+website: https://www.london.droidcon.com/
 location: London, UK
 
 date_start: 2020-10-29
 date_end:   2020-10-30
 
 cfp_start: 2019-11-08
 cfp_end:   2020-05-30
-cfp_site: http://uk.droidcon.com/ # the CFP link is broken
+cfp_site: https://sessionize.com/droidcon-sf-2020/
```

Comment:
SF?

Reply:
It's weird but that's the link given on the website. 
The link does have the DC London banner and venue. Seems like they are re-using the page